src/helpers/beams.ts

```diff
@@ -2,12 +2,18 @@ import * as PusherPushNotifications from '@pusher/push-notifications-web';
 
 let beams: any;
 
-try {
-  beams = new PusherPushNotifications.Client({
-    instanceId: (import.meta.env.VITE_PUSHER_BEAMS_INSTANCE_ID as string) ?? ''
-  });
-} catch (e) {
-  console.log(e);
-}
+(async () => {
+  try {
+    window.navigator.serviceWorker.register(import.meta.env.VITE_BASE_PATH + 'service-worker.js');
+    const serviceWorkerRegistration = await window.navigator.serviceWorker.ready;
+  
+    beams = new PusherPushNotifications.Client({
+      instanceId: (import.meta.env.VITE_PUSHER_BEAMS_INSTANCE_ID as string) ?? '',
+      serviceWorkerRegistration
+    });
+  } catch (e) {
+    console.log(e);
+  }
+})();
```

Comment:
Beams expects a `service-worker.js` to be at `/` if you don't provide your own one.

Reply:
ok with the change if you tested that push notification works fine.